packages/connect/src/api/cardanoGetPublicKey.ts

```diff
@@ -7,10 +7,11 @@ import { getMiscNetwork } from '../data/coinInfo';
 import { validatePath, fromHardened, getSerializedPath } from '../utils/pathUtils';
 import { UI, createUiMessage } from '../events';
 
-export default class CardanoGetPublicKey extends AbstractMethod<
-    'cardanoGetPublicKey',
-    PROTO.CardanoGetPublicKey[]
-> {
+interface Params extends PROTO.CardanoGetPublicKey {
+    suppressBackupWarning?: boolean;
```

Comment:
just thinking out loud maybe it would be better to add it in to `CommonParams`?

Reply:
It could be there. In common params, there are already params that are not trully common (meaning a param that can affect every method). It is probably better to move it to common 🤔